### DISCUSSION TITLE
Wargroove

### DISCUSSION THREAD

**Discussion by Heliumrich** [skyline-emu/skyline#448] [Game Requests]
A nice 2D turn based game, inspired by Advance Wars

I can provide a log if needed

Thanks <3

**Comment by iniMerchant64** [comment]
That would count towards unity games. Vote for that in the other request i made. Same with katana zero

**Reply by Heliumrich** [reply]
Nah, they use their own game engine afaik

https://github.com/amzeratul/halley

**Reply by Heliumrich** [reply]
And Katana Zero is made with Game Maker Studio 2

Do a quick google search next time

**Reply by iniMerchant64** [reply]
I did. Google lied to me
